### PR TITLE
Recdeb eventgrid

### PR DESCRIPTION
**Pull request by mscaldas2012**

### CODE REVIEW

File: fns-hl7-pipeline/fn-receiver-debatcher/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>
         <gson.version>2.10.1</gson.version>
         <hl7pet.version>1.2.7.1</hl7pet.version>
-        <lib-dex-commons.version>0.0.39-SNAPSHOT</lib-dex-commons.version>
+        <lib-dex-commons.version>0.0.41-SNAPSHOT</lib-dex-commons.version>
```
Comment (marcia-schulman) [inline fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:28]: I thought we were naming the versions according to the Sprint release number? 0.0.41 is a few weeks away.
Is it possible to add a minor version to the end instead? 0.0.39.1 ?

Comment (mscaldas2012) [reply fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:28]: libraries don't have to follow the release number - i had to release that twice on a single sprint and if i dn't change the version, you can get caching problems!

Comment (mscaldas2012) [reply fns-hl7-pipeline/fn-receiver-debatcher/pom.xml:28]: just noticed your suggestion but no - we are following SEM version and there's no 4th number :)

File: fns-sidecars/fn-sc-processapi-report/pom.xml [hunk 2]
```diff
@@ -147,10 +147,10 @@
                     <appServicePlanName>ocio-ede-${environment-id}-hl7-elastic-service-plan6</appServicePlanName>
                     <region>${functionRegion}</region>
                     <subscriptionId>${subscriptionID}</subscriptionId>
-                    <auth>
+                    <!--<auth>
```
Comment (marcia-schulman) [inline fns-sidecars/fn-sc-processapi-report/pom.xml:150]: Please remove comments from auth tags

**Review by marcia-schulman — CHANGES_REQUESTED** [review]

**Review by marcia-schulman — APPROVED** [review]

**Pull request merged**